types/shelljs/index.d.ts

```diff
@@ -537,10 +544,28 @@ export interface ShellConfig {
     fatal: boolean;
 
     /**
+     * Will print each executed command to the screen. Default is true.
+     * @type {boolean}
+     */
+    verbose: boolean;
+
+    /**
+     * Passed to glob.sync() instead of the default options ({}).
+     * @type {Object}
+     */
+    globOptions: GlobOptions;
+
+    /**
      * Absolute path of the Node binary. Default is null (inferred).
      * @type {string|null}
      */
     execPath: string | null;
+
+    /**
+     * Reset shell.config to the defaults.
+     * @type {ResetConfigFunction}
+     */
+    reset: ResetConfigFunction;
```

Comment:
You could just use JSDoc to annotate the function parameters (none in this case) and the return type (`void`). No need for a `@type` in this case.
```
@returns {void}
```